scripts/npm/create-component.js

```diff
@@ -165,6 +170,43 @@ const createComponent = async (err, { name }) => {
 
     fs.writeFileSync(storybookConfigFile, newStorybookConfigContent, 'utf8');
 
+    // Add the new component to .spelling:
+    const spellingFileContent = fs.readFileSync(spellingFile).toString();
+
+    const spellingFileComponents = spellingFileContent
+      .split(SPELLING_COMPONENTS_SPLIT_POINT_1)[1]
+      .split(SPELLING_COMPONENTS_SPLIT_POINT_2)[0]
+      .split('\n')
+      .filter(s => !_.isEmpty(s));
+
+    spellingFileComponents.push(`Bpk${pascalCaseName}`);
+
+    const spellingFilePackages = spellingFileContent
+      .split(SPELLING_PACKAGES_SPLIT_POINT_1)[1]
+      .split(SPELLING_PACKAGES_SPLIT_POINT_2)[0]
+      .split('\n')
+      .filter(s => !_.isEmpty(s));
+
+    spellingFilePackages.push(`bpk-${name}`);
+
+    let newSpellingFileContent = `${
+      spellingFileContent.split(SPELLING_COMPONENTS_SPLIT_POINT_1)[0]
+    }${SPELLING_COMPONENTS_SPLIT_POINT_1}\n${spellingFileComponents
```

Comment:
This logic could do with some refactoring now that we're using it 3 times!

Reply:
Go on then 😉

Reply:
I did 😜